addons/survey/views/survey_templates.xml

```diff
@@ -185,10 +185,21 @@
             t-att-data-timer="timer_start"
             t-att-data-time-limit-minutes="time_limit_minutes"/>
         <t t-if="survey.questions_layout == 'one_page'">
-            <t t-foreach='survey.question_and_page_ids' t-as='question'>
-                <h2 t-if="question.is_page" t-field='question.title' class="o_survey_title pb16 text-break w-lg-50 mx-lg-auto" />
-                <div t-if="question.is_page" t-field='question.description' class="text-break w-lg-50 mx-lg-auto"/>
-                <t t-if="not question.is_page and question in answer.predefined_question_ids" t-call="survey.question_container"/>
+            <!-- Questions without section -->
```

Comment:
I have a question here. I feel like we are creating two chunks: One chunk for the questions with no section and one chunk with a section. Wouldn't it change the display order of the questions?

Reply:
If it becomes too complex and we are not sure about the impact of the change, we can maybe keep the same structure as before but instead of fetching the parent section with `closest`, we select the sibling section of the question (with a loop and `previousSibling`).